owncloud/core
# Allow search in synced addressbooks

# Test Scenario 1
1. make sure the local system addressbook has been generated (occ command)
2. search for a local user in the share tab (yes - local remote shares are stupid - this is for testing)
   note that search is based on the display name
3. see the local user being displayed in the drop down
# Test Scenario 2
1. have two instances trusting each other
2. have local addressbooks generated
3. have them remote addressbooks synced
4. seach for a user on the other system within the share tab
5. see the remote user popping up
